tests/test_admin.py

```diff
@@ -0,0 +1,18 @@
+from subprocess import call
+
+
+from config import Config
+from models.users import User
+
+
+def test_create_admin():
+    call(['flask', 'create_admin'])
+    email_list = Config.ADMIN_LIST.split()
+    check = []
+    for email in email_list:
+        user = User.query.filter_by(email=email).first()
+        try:
+            check.append(user.has_role('ADMIN'))
+        except AttributeError:
+            assert False, 'Admin accounts not created'
+    assert all(check)

```

Comment:
`all()` returns `True` also for empty list and we want assert in this test that there will be some admins created.
So either split this test to two different cases, or change the assertion.